towhee/config/log_config.py

```diff
@@ -0,0 +1,47 @@
+# Copyright 2021 Zilliz. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+from typing import Optional, Any
+
+from pydantic import BaseModel, validator
+from towhee.utils.singleton import singleton
+
+
+@singleton
+class LogConfig(BaseModel):
```

Comment:
If this is a user-facing interface, more detailed comments are needed, including parameter descriptions, examples, etc.